dir.props

```diff
@@ -88,18 +101,6 @@
     <HostPolicyVersion Condition="'$(UseShippedHostPolicyPackage)' == 'true'">2.0.0</HostPolicyVersion>
   </PropertyGroup>
 
-  <!-- list of nuget package sources passed to dnu -->
-  <ItemGroup>
-    <!-- Need to escape double forward slash (%2F) or MSBuild will normalize to one slash on Unix. -->
-    <DotnetSourceList Include="$(OverridePackageSource)" />
```

Comment:
Don't you still need a replacement for the DotnetSourceList somewhere?  I seem to be missing the connection between the RestoreSources property above, the DotnetSourceLIst here and the use of DotnetRestoreSource below.

Reply:
Nevermind I found the missing link. https://github.com/dotnet/buildtools/pull/1707

Reply:
The advantage with this approach is that I can now open https://github.com/dotnet/core-setup/blob/master/src/managed/Microsoft.Extensions.DependencyModel/Microsoft.Extensions.DependencyModel.csproj in VS 2017, and restore works correctly.